platforms/fabric/src/main/java/com/dfsek/terra/fabric/FabricPlatform.java

```diff
@@ -17,43 +17,24 @@
 
 package com.dfsek.terra.fabric;
 
-import ca.solostudios.strata.Versions;
-import ca.solostudios.strata.parser.tokenizer.ParseException;
-import ca.solostudios.strata.version.Version;
 import net.fabricmc.loader.api.FabricLoader;
 import org.jetbrains.annotations.NotNull;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
 
 import java.io.File;
 import java.util.Collection;
 import java.util.stream.Collectors;
-import java.util.stream.Stream;
 
-import com.dfsek.terra.addon.EphemeralAddon;
 import com.dfsek.terra.api.addon.BaseAddon;
 import com.dfsek.terra.lifecycle.LifecyclePlatform;
 
 
 public class FabricPlatform extends LifecyclePlatform {
-    private static final Logger LOGGER = LoggerFactory.getLogger(FabricPlatform.class);
     
     @Override
     protected Collection<BaseAddon> getPlatformMods() {
-        return FabricLoader.getInstance().getAllMods().stream().flatMap(mod -> {
-            String id = mod.getMetadata().getId();
-            if(id.equals("terra") || id.equals("minecraft") || id.equals("java")) return Stream.empty();
-            try {
-                Version version = Versions.parseVersion(mod.getMetadata().getVersion().getFriendlyString());
-                return Stream.<BaseAddon>of(new EphemeralAddon(version, "fabric:" + id));
-            } catch(ParseException e) {
-                LOGGER.warn(
-                        "Mod {}, version {} does not follow semantic versioning specification, Terra addons will be unable to depend on " +
-                        "it.",
-                        id, mod.getMetadata().getVersion().getFriendlyString());
-            }
-            return Stream.empty();
-        }).collect(Collectors.toList());
+        return FabricLoader.getInstance().getAllMods().stream().flatMap(
+                mod -> parseModData(mod.getMetadata().getId(), mod.getMetadata().getVersion().getFriendlyString(), "fabric")).collect(
```

Comment:
Change to the following:
```java
return FabricLoader.getInstance().getAllMods().stream().flatMap(
        mod -> parseModData(mod.getMetadata().getId(), mod.getMetadata().getVersion().getFriendlyString(), "fabric")
).collect(Collectors.toList());
```

Reply:
this should be fixed but changing the auto formatter @Astrashh

Reply:
afaik this isn't an autoformatter issue, because the autoformatter will still sometimes respect the manually added newlines. and the autoformatter isn't intelligent enough to fix such a case.

Reply:
IntelliJ actually will not let me do that
![image](https://github.com/PolyhedralDev/Terra/assets/25417900/763fb945-b609-4fcb-99fb-94ec0cc6b439)
this is what happens when I try to paste in that

Reply:
it didn't look like what you posted so I wanted to make sure

Reply:
idc if it looks exactly like that, I just wanted the `collect` method to be on the same line as its arguments & brackets

Reply:
👍